pkg/littlefs/Makefile.littlefs

```diff
@@ -0,0 +1,5 @@
+MODULE := littlefs
+
+CFLAGS += -Wno-unused-parameter -Wno-format
```

Comment:
mhm, might be worth upstreaming some fixing then 😉

Reply:
Yeah, we'll see later ^^